src/view/items/Yielder.js

```diff
@@ -42,6 +43,14 @@ export default class Yielder extends Item {
 		}).bind();
 	}
 
+	bubble () {
+		if ( !this.dirty ) {
+			runloop.addFragment( this.fragment );
```

Comment:
I'm pretty sure you're right. I'll merge this and verify that the original fiddle still passes.